src/resolver/didValidator.ts

```diff
@@ -92,6 +95,7 @@ function getCredoTsDidResolver(agentContext: AgentContext): Resolver {
 }
 
 /**
+ * @deprecated This function is deprecated and will be removed in an upcoming version.
```

Comment:
Remove it now if you want. We are already doing breaking changes and I don't think there are apps relying on this method. Are they?